src/rust/lib/enso-prelude/src/option.rs

```diff
@@ -6,6 +6,7 @@ pub trait OptionOps {
     fn map_ref      <U,F> (&self , f:F) -> Option<U> where F : FnOnce(&Self::Item) -> U;
     fn for_each     <U,F> (self  , f:F)              where F : FnOnce(Self::Item)  -> U;
     fn for_each_ref <U,F> (&self , f:F)              where F : FnOnce(&Self::Item) -> U;
+    fn map_none     <F>   (self  , f:F) -> Self      where F : FnOnce();
```

Comment:
Why this is called "map"? It does not map anything. It's rather something like "if none"

Reply:
For symmetry with `Result` and `map_err`. Imagine that this function maps one `None` into another. As `None` is just a single value type, it is collapsed to void.